packages/pn-personafisica-login/src/pages/loginError/LoginError.tsx

```diff
@@ -94,7 +93,7 @@ const LoginError = () => {
           {getErrorMessage()}
         </Typography>
         <Button id="login-button" variant="contained" onClick={goToLogin}>
-          {getLocalizedOrDefaultLabel('common', 'button.go-to-login', 'Accedi')}
+          {t('loginPage.loginBox.title')}
```

Comment:
Why do we change button.go-to-login into loginPage.loginBox.title?

Reply:
Ok, I've restored button.go-to-login and added the translations for every language.
@ZorbaDimatteo , I found the translations on pn-pa-webapp.